features/admin.connections.v1/components/authenticator-grid.tsx

```diff
@@ -254,13 +280,26 @@ export const AuthenticatorGrid: FunctionComponent<AuthenticatorGridPropsInterfac
      *
      * @param id - Authenticator ID.
      */
-    const handleAuthenticatorDelete = (id: string): void => {
+    const handleAuthenticatorDelete = (id: string, connectionType: string): void => {
 
         setIsDeletionLoading(true);
 
-        deleteConnection(id)
+        let deleteAction: (id: string) => Promise<AxiosResponse>;
+
+        switch (connectionType) {
+            case ConnectionTypes.IDVP:
+                deleteAction = deleteIdentityVerificationProvider;
+
+                break;
+
+            default:
+                deleteAction = deleteConnection;
+
+                break;
+        }
```

Comment:
```suggestion
        let deleteAction: (id: string) => Promise<AxiosResponse> = connectionType === ConnectionTypes.IDVP
            ? deleteIdentityVerificationProvider;
            : deleteConnection;
```

Reply:
IMO, switch case is more readable than the ternary syntax.
And when there are more connection types listed on the connections page (which is a future plan), it'll be more convenient to extend the impl with switch case.